internal/consensus/common_test.go

```diff
@@ -68,8 +67,15 @@ func ensureDir(t *testing.T, dir string, mode os.FileMode) {
 	require.NoError(t, tmos.EnsureDir(dir, mode))
 }
 
-func ResetConfig(dir, name string) (*config.Config, error) {
-	return config.ResetTestRoot(dir, name)
+func ResetConfig(t *testing.T, dir, name string) (*config.Config, error) {
```

Comment:
we can simplify a list of arguments: to pass a `dir` argument into this function, we alway call `t.TempDir()`, so we can do it internally